site/ko/tutorials/eager/automatic_differentiation.ipynb

```diff
@@ -0,0 +1,322 @@
+    "초기값으로 GradientTape.gradient() 메서드가 호출되면 GradientTape에 포함된 리소스가 해체되게 설정돼있습니다. 동일한 연산을 통해서 여러 그래디언트를 계산하려면, `지속성있는(persistent)` 그래디언트 테이프를 생성하면 됩니다. `persistent`는 `gradient()` 메서드의 다중 호출을 허용합니다. 테이프 객체가 쓰레기 수집(garbage collection)될때 리소스는 해체됩니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "zZaCm3-9zVCi"
+   },
+   "outputs": [],
+   "source": [
+    "x = tf.constant(3.0)\n",
+    "with tf.GradientTape(persistent=True) as t:\n",
+    "  t.watch(x)\n",
+    "  y = x * x\n",
+    "  z = y * y\n",
+    "dz_dx = t.gradient(z, x)  # 108.0 (4*x^3 at x = 3)\n",
+    "dy_dx = t.gradient(y, x)  # 6.0\n",
+    "del t  # 리소스가 해체됩니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "6kADybtQzYj4"
+   },
+   "source": [
+    "### 제어흐름(Control Flow) 기록\n",
+    "\n",
+    "테이프를 실행하는 순간부터 연산을 기록하기 때문에, 파이썬 제어흐름(예를 들어 `if` `while`, `for`문 같은)은 자연스럽게 처리됩니다. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "9FViq92UX7P8"
+   },
+   "outputs": [],
+   "source": [
+    "def f(x, y):\n",
+    "  output = 1.0\n",
+    "  for i in range(y):\n",
+    "    if i > 1 and i < 5:\n",
+    "      output = tf.multiply(output, x)\n",
+    "  return output\n",
+    "\n",
+    "def grad(x, y):\n",
+    "  with tf.GradientTape() as t:\n",
+    "    t.watch(x)\n",
+    "    out = f(x, y)\n",
+    "  return t.gradient(out, x) \n",
+    "\n",
+    "x = tf.convert_to_tensor(2.0)\n",
+    "\n",
+    "assert grad(x, 6).numpy() == 12.0\n",
+    "assert grad(x, 5).numpy() == 12.0\n",
+    "assert grad(x, 4).numpy() == 4.0\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "DK05KXrAAld3"
+   },
+   "source": [
+    "### 고차원(Higher-order) 그래디언트\n",
+    "\n",
+    "`GradientTape` 컨텍스트 매니저안에 있는 연산들은 자동미분을 위해 기록됩니다. 만약 그래디언트가 컨텍스트 안에서 연산되면 그래디언트 연산 또한 기록되어집니다. 그 결과 똑같은 API가 고차원 그래디언트에서도 잘 작동합니다. 예를 들면:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "cPQgthZ7ugRJ"
+   },
+   "outputs": [],
+   "source": [
+    "x = tf.Variable(1.0)  # 1.0으로 초기화된 텐서플로 변수 생성\n",
+    "\n",
+    "with tf.GradientTape() as t:\n",
+    "  with tf.GradientTape() as t2:\n",
+    "    y = x * x * x\n",
+    "  # t 컨텍스트 매니저 안의 그래디언트 연산\n",
+    "  # 이것은 또한 그래디언트 계산이 미분가능하다는것을 의미합니다. \n",
+    "  dy_dx = t2.gradient(y, x)\n",
+    "d2y_dx2 = t.gradient(dy_dx, x)\n",
+    "\n",
+    "assert dy_dx.numpy() == 3.0\n",
+    "assert d2y_dx2.numpy() == 6.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "4U1KKzUpNl58"
+   },
+   "source": [
+    "## 다음 단계\n",
+    "\n",
+    "이번 튜토리얼에서는 텐서플로에서 그래디언트 계산법을 배웠습니다. 이를 통해 우리는 신경망을 구축하고 훈련시키는 데 필요한 기본 요소를 충분히 확보할 수 있습니다."
```

Comment:
To translate 'cover',
'배웠습니다' --> '다루었습니다'

Remove '우리는'.

You can use '배웠습니다' here because it is duplicated.
'enough' can be translated like 'many'.
'기본 요소를 충분히 확보할 수 있습니다' --> '많은 기본 요소를 배웠습니다'